src/Cohort.js

```diff
@@ -0,0 +1,30 @@
+class Cohort {
+  constructor(cohortName, capacity = 24) {
+    this.cohortName = cohortName
+    this.studentInside = []
+    this.capacity = capacity
+  }
+
+  addStudent(student) {
+    this.studentInside.push(student)
+    return `${student.name} is now inside Cohort ${this.cohortName}`
```

Comment:
I'd probably just return the student itself rather than a message. It's also okay to not return anything at all